### PR TITLE
Make sure all @license comments are maintained when flattening

### LINKED ISSUE #308: @license comments missing from imports
Because the inlining only copies the contents of `<head>` and `<body>`, comments with `@license` after the doctype will be missing in the output.

### PR DESCRIPTION
**Pull request by azakus**
Fixes #308

### CODE REVIEW

**Comment by azakus** [comment]
PTAL @garlicnation or @justinfagnani

File: lib/vulcan.js [hunk 1]
```diff
@@ -183,6 +183,22 @@ Vulcan.prototype = {
...
 
+  isLicenseComment: function(node) {
+    if (dom5.isCommentNode(node)) {
+      return dom5.getTextContent(node).indexOf('@license') > -1;
+    }
+    return false;
+  },
+
+  collectLicenseComments: function(nodes, collection) {
+    for (var i = 0, n; i < nodes.length; i++) {
```
Comment (justinfagnani) [inline lib/vulcan.js]: Could be `Array.push.apply(collection, nodes.filter(isLicenseComment));`

Comment (justinfagnani) [inline lib/vulcan.js]: Or always return a new array and `concat()` them in `flatten`

**Comment by justinfagnani** [comment]
LGTM

**Pull request merged**